crates/zk-por-core/src/merkle_sum_tree.rs

```diff
@@ -62,7 +69,7 @@ impl MerkleSumTree {
             }
         }
 
-        MerkleSumTree { merkle_sum_tree, tree_depth: log2_strict(accounts.len()) }
+        MerkleSumTree { merkle_sum_tree, tree_depth }
     }
 
     pub fn get_root(&self) -> MerkleSumNode {
```

Comment:
can add a test case to assert that the generated root is identical to the one in generated in plonky2 proof.